### ISSUE TITLE
Format specifiers for inline-memory watch windows 'unable to read process memory'

### ISSUE THREAD

**Issue by JustSlavic** [x13pixels/remedybg-issues#255]
When I allocate memory with malloc, for example:

```c++
int main() {
    int size = 10;
    void *p = malloc(size);
    // ...
    free(p);
    return 0;
} 
```

I can view it in the watch window as an array `p, size` and `p, dd` work just fine, but view as `p, dd size` says 'unable to read process memory':
![image](https://user-images.githubusercontent.com/26839955/178783178-b9fad826-c3fe-415b-8222-d5d465a74c44.png)

Also, if I click on the magnifying glass icon, the popup window works, and updates as you would expect, but all memory always red.
![image](https://user-images.githubusercontent.com/26839955/178786506-2ef161ab-4ed6-410e-b516-b3e60da3dbeb.png)

**Comment by x13pixels** [comment]
The `,dc` formatter, and the other memory formatters, take an ending _address_ rather than size. Here the expression would be `p, dc (char*)p + size`. Probably should be consistent with the array format and / or clearer here.

The discrepancy in the popup is definitely wrong and will leave this issue opened for handling it. Thanks.

**Comment by JustSlavic** [comment]
I misread the README indeed, I'm sorry. I even tried to have formattings like `p, (char*)p + size`, but they didn't work neither, because I didn't use formatters with them. Oops.
I just thought for some reason that they would work similarly or either way.
Thank you for clarification!

**Comment by x13pixels** [comment]
Fixed the discrepancy in 0.3.9.9 where the watch window would show "??? (unable to read process memory" but the separated window would show invalid memory contents. They will now both show "??? (unable to read process memory" if the target memory cannot be read.
